app/models/street/StreetEdgeTable.scala

```diff
@@ -288,31 +288,68 @@ object StreetEdgeTable {
 
   /** Gets a list of all street edges that the user has audited */
   def selectAllStreetsAuditedByAUser(userId: UUID): List[StreetEdge] = db.withSession { implicit session =>
-    val selectAuditedStreetsQuery = Q.query[String, StreetEdge](
-      """SELECT street_edge.street_edge_id, street_edge.geom, source, target, x1, y1, x2, y2, way_type, street_edge.deleted, street_edge.timestamp
-        |  FROM sidewalk.street_edge
-        |INNER JOIN sidewalk.street_edge_region
-        |  ON street_edge_region.street_edge_id = street_edge.street_edge_id
-        |INNER JOIN sidewalk.audit_task
-        |  ON street_edge.street_edge_id = audit_task.street_edge_id
-        |  AND audit_task.completed = TRUE
-        |  AND audit_task.user_id = ?
-        |WHERE street_edge.deleted=FALSE
-      """.stripMargin
-    )
-    selectAuditedStreetsQuery(userId.toString).list.groupBy(_.streetEdgeId).map(_._2.head).toList
+    selectAllStreetsAuditedByAUserQuery(userId).list
+  }
+
+  /** Gets the query for a list of all street edges that the user has audited */
+  def selectAllStreetsAuditedByAUserQuery(userId: UUID) = db.withSession { implicit session =>
+
+    val auditedStreets = for {
+      (_streetEdges, _auditTasks) <- streetEdgesWithoutDeleted.innerJoin(completedAuditTasks).on(_.streetEdgeId === _.streetEdgeId)
+      if _auditTasks.userId === userId.toString
+    } yield _streetEdges
+    auditedStreets.groupBy(x => x).map(_._1) // does a select distinct
   }
```

Comment:
Rewrote the query using Scala/slick. But I also removed that inner join with the street_edge_region table b/c it wasn't doing anything. 

If you want a long-winded explanation of how it is not used, then here you go:
The street_edge_region table just has entries that pair every street edge with each neighborhood that it intersects. So most edges have 1 entry, and some have 2 (idk about more than that). And all street edges are in regions so its not like the inner join filters any of them out. It basically just adds a duplicate for each street edge that intersects multiple regions, then none of the columns in that table are used, and a select distinct is essentially done on the street_edge_id anyway.